src/main/java/org/junitpioneer/jupiter/params/DisableIfDisplayName.java

```diff
@@ -16,7 +16,6 @@
 import java.lang.annotation.Target;
```

Comment:
Please revert those changes - they have nothing to do with the topic of this PR!

Reply:
actually they have :D because DisplayIf was removed in 5.6.0 and due to that the build fails - i thought i will just fix this with this build

Reply:
Oh okay :)